src/api.js

```diff
@@ -2399,6 +2400,7 @@ api.declare({
       workerId,
       recentTasks: Entity.types.SlugIdArray.create(),
       expires: expires || taskcluster.fromNow('1 day'),
+      disabled: false,
       firstClaim: new Date(),
     });
```

Comment:
You'll need to catch the case there the entity exists... in case you are racing...

Note: I might be tempting to make a `this.Worker.upsert` method in `data.js` that does the right thing in these cases... Assuming it can be easily defined what that is for the `Worker` table..

Reply:
Sounds good